portal/src/texts/guider/sass-stilguide.mdx

```diff
@@ -0,0 +1,201 @@
+
+Begrepene _Sass_ og _SCSS_ blir brukt om hverandre i Jøkul, og betyr enten Sass-prosjektet eller Sass-kode skrevet med SCSS-syntaks.
+
+## Block Element Modifier
+
+Vi bruker metodikken [Block Element Modifier](https://css-tricks.com/bem-101/) (BEM) i Jøkul.
+
+Nøsting og [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector) i Sass gjør det enklere å skrive CSS på denne måten.
+
+```scss
+.block {
+    /* ... */
+    &__element {
+        /* ... */
+        &--element-modifier {
+            /* ... */
+        }
+    }
+
+    &--block-modifier {
+        /* ... */
+    }
+}
+```
+
+### Flere elementnivåer
+
+Dersom du er i ferd med å skrive en `block__element__another-element`-selector, stopp og tenk om `element` egentlig er en _annen_ `block`.
+
+## Moduler
+
+I Jøkul bruker vi bare [modul-syntaks](https://sass-lang.com/blog/the-module-system-is-launched) med [@use](https://sass-lang.com/documentation/at-rules/use) og [@forward](https://sass-lang.com/documentation/at-rules/forward), ikke `@import`. Hver fil er en modul, på samme måte som du kanskje kjenner til fra [TypeScript](/stilguide/react).
+
+Moduler bør holdes til ett tema. Et tema kan for eksempel være CSSen for en komponent, mixins som har å gjøre med skjermstørrelser, eller funksjoner for konvertering av data.
+
+### Partials
+
+Vi bruker [partials](https://sass-lang.com/guide#topic-4) (filer hvor navnet starter med `_`) for alle filer som ikke skal kompileres direkte til CSS. Eksempler er moduler som består utelukkende av variabler, funksjoner og mixins. Det kan også være en modul med CSS for en komponent, men hvor alt skal samles opp i én enkelt CSS-fil til slutt (se for eksempel [packages/table](https://github.com/fremtind/jokul/tree/main/packages/table)).
+
+### Mappe- og filstruktur
+
+Hver mappe med Sass skal ha en `_index.scss` hvor det listes opp `@forward` for alle filer i mappa.
+
+Et unntak er på rotnivå i pakkene (for eksempel `packages/table/_index.scss`). Her skal det være én `@forward` som peker på hovedfila (`table.scss` i dette tilfellet), og så har _den_ fila ansvaret for å hente inn alt som skal være med. Det gjør vi for at Sass skal finne alle avhengighetene til `table.scss` når den kompileres til `table.css`.
+
+### Offentlig API
+
+Mappenavn og filnavn er **ikke** en del av APIet til en pakke og kan endres uten forvarsel, med unntak av:
+
+-   Filnavnet som matcher pakkens navn, for eksempel `core.scss` og `table.scss`. Disse navnene brukes av prosjekter som importerer CSS.
+-   `packages/core/_jkl.scss`, som vi ser på som "hovedinngangen" til Sass-verktøy som mixins, funksjoner og variabler i `core`
+
+## Variabler
+
+Med unntak av i `packages/core` skal variabler som regel være _private_. Private variabler kan ikke brukes av andre moduler. Varibler er _private_ om navnet starter med `_` (underscore).
```

Comment:
Er litt inkonsekvent med bruken av `em` for `private` og `public`. Tenker jeg kan fjerne alt og bare ha det som vanlig brødtekst (ikke kursiv).